templates/controllers/grid/settings/roles/form/userGroupForm.tpl

```diff
@@ -16,6 +16,7 @@
 			selfRegistrationRoleIds: {$selfRegistrationRoleIds|@json_encode},
 			recommendOnlyRoleIds: {$recommendOnlyRoleIds|@json_encode},
 			roleForbiddenStagesJSON: {$roleForbiddenStagesJSON},
+			notChangeMetadataEditPermissionRoles: {$notChangeMetadataEditPermissionRoles|@json_encode},
```

Comment:
`@` in this case is subtle :)

Reply:
I think I followed exactly the same approach as the `selfRegistrationRoleIds` variable here - (and I am not sure what "subtle" means here :) )

Reply:
It is correct this way -- and needed because of a nasty quirk of Smarty syntax.